contentcuration/contentcuration/dev_settings.py

```diff
@@ -1,8 +1,23 @@
 # flake8: noqa
+from dotenv import load_dotenv
+
 from .not_production_settings import *  # noqa
 
 DEBUG = True
 
 ROOT_URLCONF = "contentcuration.dev_urls"
 
 INSTALLED_APPS += ("drf_yasg",)
+
+REST_FRAMEWORK = {
+    'DEFAULT_PERMISSION_CLASSES': (
+        'rest_framework.permissions.IsAuthenticated',
+    ),
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        'rest_framework.authentication.SessionAuthentication',
+        'rest_framework.authentication.BasicAuthentication',
+        'rest_framework.authentication.TokenAuthentication',
+    )
+}
```

Comment:
To allow basic authentication during development. Btw I am also fine with not having it enabled if we think its not needed or helpful.

Reply:
I'm just wondering if this could produce differences that confuse developers when things work locally, but don't work the same way when deployed. Ideally, the differences between environment configuration/settings should be kept to a minimum.

Reply:
Umm, yup I would agree to that. Then maybe we should add some documentation around how to send authenticated requests using the TokenAuth or SessionAuth..?

Reply:
I'm not sure that adding documentation will help prevent issues here. I think it's best to remove it. If someone wants to customize behavior like this, they can use your .env functionality

Reply:
Yes, I will remove those